tests/Unit/Listeners/SendVoucherHistoryEmailTest.php

```diff
@@ -42,23 +43,23 @@ protected function setUp()
 
         // Set up voucher states.
         Auth::login($this->user);
+
         foreach ($this->vouchers as $v) {
             $v->applyTransition('order');
             $v->applyTransition('print');
             $v->applyTransition('dispatch');
             $v->applyTransition('allocate');
             $v->trader_id = 1;
             $v->applyTransition('collect');
+            $v->applyTransition('confirm');
```

Comment:
They all need to be confirmed otherwise `$voucher->paymentPendedOn` won't be defined.